### PR TITLE
Make the panels not collapsible on the experimental navigation screen

### PR DESCRIPTION
**Pull request by adamziel**
## Description

This PR makes the panels on the experimental navigation screen non-collapsible.

_"A picture is worth a thousand words"_ 

Before:

Zrzut ekranu 2020-06-1 o 12 33 44

After:

Zrzut ekranu 2020-06-1 o 12 34 07

## How has this been tested?
1. Enable the navigation experiment in Gutenberg > Experiments
1. Go to Gutenberg > Navigation (beta)
1. Confirm the panels are not collapsible

## Types of changes
New feature (non-breaking change which adds functionality)

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate. 
- [x] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

File: packages/components/src/panel/README.md [hunk 1]
```diff
@@ -127,6 +127,14 @@ An icon to be shown next to the `PanelBody` title.
 -   Type: `String`
 -   Required: No
 
+###### collapsible
```
Comment (youknowriad) [inline packages/components/src/panel/README.md:130]: Why not just use the  "Card" component, isn't it suited for these use-cases?

**Comment by tellthemachines** [comment]
Those panels were made collapsible so that people on a mobile or other small screen aren't forced to scroll through the navigator to get to the block section. Saving and deleting can currently only be done in that section, so I don't think we should remove this until we have a good solution for mobile.

**Comment by adamziel** [comment]
@tellthemachines thank you for context! How about making only the navigation panel collapsible, and only on mobile?

**Comment by adamziel** [comment]
I followed up with https://github.com/WordPress/gutenberg/pull/22828 and https://github.com/WordPress/gutenberg/pull/22827

**Pull request closed without merging**